test/unit/Resource/ParameterBag.test.js

```diff
@@ -1,8 +1,9 @@
 import ParameterBag from '../../../src/Resource/ParameterBag';
+import { setState } from '../../../src/State/State';
 
-describe('The ParameterBag', () => {
+describe('ParameterBag', () => {
```

Comment:
Qua test benaming zou ik toch gaan voor een 'The ParameterBag'. Dat dan wel combineren met it('....').